sympy/stats/tests/test_continuous_rv.py

```diff
@@ -188,12 +188,25 @@ def test_arcsin():
 
 def test_benini():
     alpha = Symbol("alpha", positive=True)
-    b = Symbol("beta", positive=True)
+    beta = Symbol("beta", positive=True)
     sigma = Symbol("sigma", positive=True)
 
-    X = Benini('x', alpha, b, sigma)
-    assert density(X)(x) == ((alpha/x + 2*b*log(x/sigma)/x)
-                          *exp(-alpha*log(x/sigma) - b*log(x/sigma)**2))
+    X = Benini('x', alpha, beta, sigma)
+    assert density(X)(x) == ((alpha/x + 2*beta*log(x/sigma)/x)
+                          *exp(-alpha*log(x/sigma) - beta*log(x/sigma)**2))
+
+    alpha = Symbol("alpha", positive=False)
```

Comment:
For me, something like this is more transparently correct:
```python
from sympy.utilities.iterables import rotate_left
npos = Symbol('np', positive=False)
args = [alpha, beta, npos]
assert [i.is_positive for i in args] == [1, 1, 0]
for _ in range(3):
    args = rotate_left(args, 1)
    raises(ValueError, lambda: Benini('x', *args)
```
YMMV

Reply:
I think a newcomer might have problem understanding this. Still, if it is better then I'll do it.
Tell me if I should change it. I'll make the commit after that